src/main/java/com/my/FoodTruckBackend/FoodTruckBackendApplication.java

```diff
@@ -1,6 +1,7 @@
 package com.my.FoodTruckBackend;
 
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
```

Comment:
I think we should remove this(Its not being used
)

Reply:
This has been resolved